docs/definitions-helpers.rst

```diff
@@ -146,4 +81,106 @@ You can also use classic xunit-style setup/teardown::
             self.assertTrue(self.container.created)
 
 
+Relationship to helpers
+^^^^^^^^^^^^^^^^^^^^^^^
+Every resource definition instance needs to have a "helper" set before it is
+possible to actually create the Docker resource that the instance defines.
+Resource helpers are described in more detail later in this section, but for
+now, know that a helper needs to be provided to the definition in one of three
+ways:
+
+1. Using the ``helper`` keyword argument in the constuctor::
+
+    helper = DockerHelper()
+    network = NetworkDefinition('net01', helper=helper)
+    network.setup()
+
+2. Using the ``helper`` keyword argument in the ``setup()`` method::
+
+    helper = DockerHelper()
+    volume = VolumeDefinition('vol02')
+    volume.setup(helper=helper)
+
+3. Directly, using the ``set_helper()`` method::
+
+    helper = DockerHelper()
+    container = ContainerDefinition('con03', 'nginx:alpine')
+    container.set_helper(helper)
+    container.setup()
+
+This only needs to be done once for the lifetime of the definition.
+
+For the most part, interaction with Docker should almost entirely occur via the
+definitions, but the definition objects need the helpers to actually interact
+with Docker.
+
+
+Mapping to Docker SDK types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Each resource definition wraps a model from the `Docker SDK for Python`_. The
+underlying model can be accessed via the ``inner()`` method, after the resource
+has been created. The mapping is as follows:
+
+===================================================  ============================================
+Seaworthy resource definition                        Docker SDK model
+===================================================  ============================================
+:class:`~seaworthy.definitions.ContainerDefinition`  :class:`docker.models.containers.Container`
+:class:`~seaworthy.definitions.NetworkDefinition`    :class:`docker.models.networks.Network`
+:class:`~seaworthy.definitions.VolumeDefinition`     :class:`docker.models.volumes.Volume`
+===================================================  ============================================
+
+Helpers
+-------
```

Comment:
This section is moved but unchanged, except for the addition of the "Mapping to Docker SDK types" subsection.